backend/src/graphql/resolver/ContributionResolver.ts

```diff
@@ -732,7 +732,7 @@ export class ContributionResolver {
 
     const event = new Event()
     const eventAdminContributionDeny = new EventAdminContributionDeny()
-    eventAdminContributionDeny.userId = contributionToUpdate.userId
+    eventAdminContributionDeny.userId = moderator.id
```

Comment:
Why this? I agree, that we should store the moderator ID as well. But this should be on xUserId

Reply:
Hmm I understand what you mean but the person who did this action is not the contribution.userId but the admin user.id that was why I changed it here and made a review on @ulfgebhardt with the remark.
But it is open to discussion if you mean that it would bring more to set contribution.userid in the event then we can remove the change and write admin.id in xUserId.

I was just a bit chocked cause I expected that admin.id is in the event.userId field.